vuepress/docs/next/tutorials/create/mfe/communication.md

```diff
@@ -265,392 +223,51 @@ const widgetEvent = new CustomEvent('greeting', {
 window.dispatchEvent(widgetEvent);
 ```
 
-The custom event should now display in the `subscriber-widget`.
+3. Confirm the custom event is displayed in the `subscriber-widget`
 
 ::: tip Congratulations!
-You’ve now created a micro frontend that listens to custom events.
+You’ve now created a micro frontend that listens to custom events
 :::
 
 ## Add Widgets to App Builder
 
-Now let's add the publisher and subscriber micro frontends in Entando.
-
-### Create Environment File
-
-#### Publisher Widget
-
-1. Create an `.env` file in the project root for the `publisher-widget`.
-
-2. Open the `.env` file, and enter the `PUBLIC_URL` where the micro frontend will be hosted.
-
-Example:
+To add the publisher and subscriber MFEs to Entando, run the following commands from the root folder of each:
```

Comment:
each reads like it's referencing publish and subscriber, not the MFEs, even though indirectly they are mfes but I was thinking what publisher directory?